sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -381,7 +381,7 @@ def test_lambertw():
     assert LambertW(2, evaluate=False).is_real
     p = Symbol('p', positive=True)
     assert LambertW(p, evaluate=False).is_real
-    assert LambertW(p - 1, evaluate=False).is_real is None
+    assert LambertW(p - 1, evaluate=False).is_real is False
```

Comment:
https://github.com/sympy/sympy/blob/master/sympy/core/add.py#L612 need to change here
i mean ```exp``` is constant not variable so we need to add in to c(constant) but still it's add with variable```a```.